.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,35 @@
+name: Renovate
+on:
+  push:
+    paths:
+      - ".github/workflows/renovate.yaml"
+      - ".github/renovate-config.js"
+  schedule:
+    - cron: "0 7 * * 5"
+  workflow_dispatch:
+
+env:
+  ACTIONS_STEP_DEBUG: ${{ secrets.ACTIONS_STEP_DEBUG }}
+
+jobs:
+  renovate:
+    runs-on: [self-hosted, linux, x64]
```

Comment:
@truongnht Sorry that I missed this, but this job will never schedule on GH-hosted runners because of this `self-hosted` requirement.  I think we'll want to schedule this on `ubuntu-latest`.